ethpm_types/base.py

```diff
@@ -33,3 +37,21 @@ def json(self, *args, **kwargs) -> str:
             kwargs["exclude_none"] = True
 
         return super().json(*args, **kwargs)
+
+    @classmethod
+    @no_type_check
+    def _get_value(
+        cls,
+        v: Any,
+        *args,
+        **kwargs,
+    ) -> Any:
+
+        if isinstance(v, HexBytes):
```

Comment:
@NotPeopling2day Is this line tested? This is what I am talking about on Tele

Reply:
Yes, when the `.dict()` method is called the value of the member will be HexBytes